gc/startup/omrgcstartup.cpp

```diff
@@ -229,15 +229,18 @@ OMR_GC_InitializeDispatcherThreads(OMR_VMThread* omrVMThread)
 omr_error_t
 OMR_GC_ShutdownDispatcherThreads(OMR_VMThread* omrVMThread)
 {
-	MM_GCExtensionsBase *extensions = MM_GCExtensionsBase::getExtensions(omrVMThread->_vm);
 	omr_error_t rc = OMR_ERROR_NONE;
 
-	if (NULL != extensions->dispatcher) {
-		extensions->dispatcher->shutDownThreads();
-		extensions->dispatcher->kill(MM_EnvironmentBase::getEnvironment(omrVMThread));
-		extensions->dispatcher = NULL;
-	}
+	if ((NULL != omrVMThread) && (NULL != omrVMThread->_vm)) {
```

Comment:
No one should ever pass a NULL OMRVMThread into this API.

Reply:
OK I wasn't sure about this one--I'll move the null check to the gc test shutdown code.